primitives/src/policy.rs

```diff
@@ -688,8 +666,6 @@ impl Default for Policy {
         Policy {
             blocks_per_batch: 60,
             batches_per_epoch: 720,
-            tendermint_timeout_init: 1000,
-            tendermint_timeout_delta: 1000,
```

Comment:
They might have been here on purpose to make them more easily configurable for tests, etc.?

Reply:
With these changes it will still be flexible as the actual values are set on the implementors of the `tendermint::Protocol` trait.

Here is an example: https://github.com/nimiq/core-rs-albatross/blob/albatross/tendermint/tests/common/mod.rs#L137-L138